lib/std/os/windows/user32.zig

```diff
@@ -1,104 +1,100 @@
 usingnamespace @import("bits.zig");
 
-// PM
+pub const WM = enum(u16) {
+    NULL = 0x0000,
+    CREATE = 0x0001,
+    DESTROY = 0x0002,
+    MOVE = 0x0003,
+    SIZE = 0x0005,
+
+    ACTIVATE = 0x0006,
+    PAINT = 0x000F,
+    CLOSE = 0x0010,
+    QUIT = 0x0012,
+    SETFOCUS = 0x0007,
+
+    KILLFOCUS = 0x0008,
+    ENABLE = 0x000A,
+    SETREDRAW = 0x000B,
+
+    SYSCOLORCHANGE = 0x0015,
+    SHOWWINDOW = 0x0018,
+
+    WINDOWPOSCHANGING = 0x0046,
+    WINDOWPOSCHANGED = 0x0047,
+    POWER = 0x0048,
+
+    CONTEXTMENU = 0x007B,
+    STYLECHANGING = 0x007C,
+    STYLECHANGED = 0x007D,
+    DISPLAYCHANGE = 0x007E,
+    GETICON = 0x007F,
+    SETICON = 0x0080,
+
+    INPUT_DEVICE_CHANGE = 0x00fe,
+    INPUT = 0x00FF,
+    KEYDOWN = 0x0100,
+    KEYUP = 0x0101,
+    CHAR = 0x0102,
+    DEADCHAR = 0x0103,
+    SYSKEYDOWN = 0x0104,
+    SYSKEYUP = 0x0105,
+    SYSCHAR = 0x0106,
+    SYSDEADCHAR = 0x0107,
+    KEYLAST = 0x0109,
+
+    COMMAND = 0x0111,
+    SYSCOMMAND = 0x0112,
+    TIMER = 0x0113,
+
+    MOUSEMOVE = 0x0200,
+    LBUTTONDOWN = 0x0201,
+    LBUTTONUP = 0x0202,
+    LBUTTONDBLCLK = 0x0203,
+    RBUTTONDOWN = 0x0204,
+    RBUTTONUP = 0x0205,
+    RBUTTONDBLCLK = 0x0206,
+    MBUTTONDOWN = 0x0207,
+    MBUTTONUP = 0x0208,
+    MBUTTONDBLCLK = 0x0209,
+    MOUSEWHEEL = 0x020A,
+    XBUTTONDOWN = 0x020B,
+    XBUTTONUP = 0x020C,
+    XBUTTONDBLCLK = 0x020D,
+};
+
+pub const WA = enum(u16) {
+    INACTIVE = 0,
+    ACTIVE = 0x0006,
+    ACTIVATE = 0x0006,
```

Comment:
What is this constant? I think you mixed up `WM_ACTIVATE`?
https://docs.microsoft.com/en-us/windows/win32/inputdev/wm-activate